cypress/e2e/tests/migration/applicationinventory/assessment/bulk_deletion_applications.test.ts

```diff
@@ -0,0 +1,64 @@
+/*
+Copyright © 2021 the Konveyor Contributors (https://konveyor.io/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+/// <reference types="cypress" />
+
+import {
+    login,
+    createMultipleApplications,
+    application_inventory_kebab_menu,
+    navigate_to_application_inventory,
+} from "../../../../../utils/utils";
+
+import { Assessment } from "../../../../models/migration/applicationinventory/assessment";
+
+let applicationList: Array<Assessment> = [];
+
+describe(["@tier2"], "Bulk deletion of applications", () => {
+    before("Login", function () {
+        login();
+    });
+
+    beforeEach("Interceptors", function () {
+        applicationList = createMultipleApplications(11);
```

Comment:
Creating 11 applications for all the tests below is not enough ?

If the list is created before each test , are all applications(33) cleared after all the tests ?

Reply:
1) Each of the tests needs at least 11 applications ( we are considering 10 items per page ).
2) The applications created for the tests get bulk deleted through the test itself, so application deletion is not required in the after() block.